src/java.desktop/share/classes/javax/swing/package-info.java

```diff
@@ -132,15 +132,13 @@
  * For overviews, tutorials, examples, guides, and other documentation,
  * please see:
  * <ul>
- *     <li><a href="http://www.oracle.com/technetwork/java/javase/tech/articles-jsp-139072.html"
- *     target="_top">The Swing Connection</a></li>
+ *     <li><a href="https://www.oracle.com/java/technologies/a-swing-architecture.html"
+ *     target="_top">A Swing Architecture Overview</a></li>
  *     <li><a href="https://docs.oracle.com/javase/tutorial/"
  *     target="_top">The Java Tutorial</a></li>
- *     <li><a href="http://www.oracle.com/technetwork/java/javase/training/index.html"
- *     target="_top">Online Training</a>
+ *     <li><a href="https://www.oracle.com/java/technologies/javase/training-support.html"
+ *     target="_top">Java SE Training and Certification</a>
  *     at the Java Developer Connection <sup>SM</sup></li>
```

Comment:
I don't have a strong opinion whether this link should be here. Maybe not, as you mentioned.

If it's kept, I think the part `at the Java Developer Connection <sup>SM</sup>` should be removed.